crates/pgls_completions/src/providers/columns.rs

```diff
@@ -717,18 +716,6 @@ mod tests {
             &pool,
         )
         .await;
-
-        // no completions in the values list!
-        assert_no_complete_results(
```

Comment:
deleted this for now; I think we should have completions in the values list, but making sure they are the right suggestions is another issue.